src/main.rs

```diff
@@ -19,12 +21,29 @@ struct Args {
     /// The OpenOCD RTT server port
     #[arg(default_value_t = 9090)]
     port: u16,
+
+    /// Write RTT messages to file
+    #[arg(short = 'o', long = "output")]
+    path: Option<String>,
 }
 
 fn main() {
     let args = Args::parse();
     let address = format!("{}:{}", args.host, args.port);
+    let output = args.path;
```

Comment:
For consistency, consider changing the name of the variable.
```suggestion
    let path = args.path;
```